lib/jobManager.js

```diff
@@ -177,7 +185,15 @@ var JobManager = module.exports = function JobManager(options) {
             options.coin.percentTreasuryReward,
             options.coin.treasuryRewardStartBlockHeight,
             options.coin.treasuryRewardAddressChangeInterval,
-            options.coin.vTreasuryRewardAddress
+            options.coin.vTreasuryRewardAddress,
+            options.coin.percentTreasuryUpdateReward,
+            options.coin.treasuryRewardUpdateStartBlockHeight,
+            options.coin.treasuryRewardUpdateAddressChangeInterval,
+            options.coin.vTreasuryRewardUpdateAddress,
+            options.coin.percentSecureNodesReward,
+            options.coin.vSecureNodesRewardAddress,
+            options.coin.percentSuperNodesReward,
+            options.coin.vSuperNodesRewardAddress
```

Comment:
(related) I think the lack of overwinter being here might be the reason we're seeing random failed VOT blocks.

Reply:
@egyptianbman lack of overwinter option being in the JobManager options?